app/src/main/java/fraktal/io/android/demo/timer/ui/TimerView.kt

```diff
@@ -77,15 +89,17 @@ private fun Render(
 }
 
 @Composable
-private fun BoxScope.RenderText(timerText: String, animateAlpha: Boolean) {
-    val alphaAnimation = remember { Animatable(1f) }
-
-    if (animateAlpha) {
-        LaunchedEffect("animateAlpha") {
-            alphaAnimation.animateTo(0.1f, animationSpec = tween(300))
-            alphaAnimation.animateTo(1f, animationSpec = tween(300))
-        }
+private fun BoxScope.RenderText(timerText: String, needAnimateAlpha: Boolean) {
+    // if build system without events - u must control your animations starts on
```

Comment:
https://github.com/jQwout/fmodel-android-demo/assets/37448471/75cccc9e-0ce7-4488-a320-48279034195d


Here you can see what happens when we get a repeat of the animation after rotating the screen.

Reply:
i alreay fix it.but it's not so easy to do this without a jetpack( for xml-ui like example)